app/views/users/show.html.erb

```diff
@@ -1,16 +1,22 @@
-<h1> Details of: <%=@user.name%> </h1>
-<p><%= @user.bio%></p>
+<ul class="pt-3">
+      <div class="container border rounded-3 p-3 col-8 justify-content-center">
+        <div class="d-inline ">
+          <img src= '<%= @user.photo %>' width="100" height="100" class="" />
+        </div>
+        <h3 class="d-inline ps-5"><%= @user.name %></h3>
+        <p class="text-end">Number of Posts: <%=@user.post_counter%></p>
+      </div>
+      <div class="container border rounded-3 p-3 col-8 justify-content-center mt-4"><%=@user.bio%></div>
+    </ul>
 
-
-<h1>All posts by  <%= @user.name %></h1>
-<p>
   <% @posts = Post.where(user_id: @user.id) %>
 </p>
 <ul>
-    <% @posts.each do |post| %>
-      <li>
-        <h4> <%= post.title %> </h4>
+    <% @posts.limit(3).each do |post| %>
+        <div class="container border rounded-3 p-3 col-8 justify-content-center mt-4">
+           <h4> <%= post.title %> </h4>
```

Comment:
- [ ]  as asked by the previous reviewer and also in the requirements you should use the methods that you created the last time to display the recent posts so you should use this [Method](https://github.com/eaesmat/blog/blob/81fc526b276dca7e5ac82dbed81ec3d13bc25295/app/models/user.rb#L8)
and try to change the code inside that method cause that code won't return the last recent posts so you change to something like `posts .order(created_at: 'desc').limit(3)`
- Screenshot of the requirements 
<img width="798" alt="Screen Shot 2022-11-03 at 15 55 20" src="https://user-images.githubusercontent.com/61790587/199757764-d8d7f0f8-d73f-418d-99f2-f95586130c77.png">